http/src/main/resources/reference.conf

```diff
@@ -392,7 +392,7 @@ gke {
 image {
   welderGcrUri = "us.gcr.io/broad-dsp-gcr-public/welder-server"
   welderDockerHubUri = "broadinstitute/welder-server"
-  welderHash = "e8828dd"
+  welderHash = "abc"
```

Comment:
Need to revert this

Reply:
oof was testing the welder sed locally and forgot to revert